packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -229,7 +230,26 @@ module.exports = async (api, args = {}, pluginSource) =>
       apisRunningByTraceId.set(apiRunInstance.traceId, 1)
     }
 
+    let stopQueuedApiRuns = false
+    let unlisten = null
```

Comment:
We think `unlisten` is  not the clearest variable name to describe the function to unsubscribe from the action we run below. Does anyone have any suggestion for a better variable name?

Reply:
I'm about to work on PR that would also need callback like this and was thinking if this should be named `onAPIRunComplete`?

I will also think I will introduce some abstraction - in this separate PR and then we can coordinate those

Reply:
So would you like me to rename the callback now or refactor it later?

Reply:
I actually needed to make changes a bit elsewhere, so I won't be touching same code. I think it makes sense to rename it here - if that name makes sense to you?

Reply:
Okay yeah the name fits. I'll push the change